.github/workflows/plugin-packages.yml

```diff
@@ -343,19 +343,19 @@ jobs:
         run: make test-log-collector-plugin-locally
 
       - name: Set up Cloud SDK
-        if: ${{ github.ref == 'refs/heads/main' }}
+        if: (needs.pre-build.outputs.logCollector == 'true') || startsWith(github.ref, 'refs/tags/')
```

Comment:
`if` not needed .. same condition is already handled at job level

```
  build_test_log-collector:
    runs-on: ubuntu-latest
    needs: pre-build
    # for tags('ref/tags/'), 'needs.pre-build.outputs.logCollector' is 'false' so to run 'build_test_log-collector' job on tags as well,
    # added condition 'startsWith(github.ref, 'refs/tags/')'
    if: (needs.pre-build.outputs.logCollector == 'true') || startsWith(github.ref, 'refs/tags/')
```

Reply:
updated